genieutils.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index b4b3abb..760a912 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -12,11 +12,11 @@ project(genieutils)
+ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/cmake/)
+ #set(CMAKE_CXX_FLAGS "-std=gnu++0x")
+ 
+-#set(Boost_USE_STATIC_LIBS        ON)
++set(Boost_USE_STATIC_LIBS        ON)
+ #set(Boost_USE_STATIC_RUNTIME     ON)
+ set(Boost_USE_MULTITHREADED      ON)
+ 
+-find_library(iconv REQUIRED)
++find_library(iconv iconv REQUIRED)
+ 
+ # dependencies:
+ 
+@@ -158,9 +158,9 @@ set(BINCOMP_SRC src/tools/bincompare/bincomp.cpp
+ # Executeable:
+ #------------------------------------------------------------------------------#
+ 
+-add_library(${Genieutils_LIBRARY} SHARED ${FILE_SRC} ${LANG_SRC} ${DAT_SRC} 
++add_library(${Genieutils_LIBRARY} STATIC ${FILE_SRC} ${LANG_SRC} ${DAT_SRC} 
+                                   ${RESOURCE_SRC} ${UTIL_SRC} ${SCRIPT_SRC} )
+-target_link_libraries(${Genieutils_LIBRARY} ${ZLIB_LIBRARIES} ${Boost_LIBRARIES} ${ICONV_LIBRARIES})
++target_link_libraries(${Genieutils_LIBRARY} ${Boost_LIBRARIES} ${ZLIB_LIBRARIES} ${ICONV_LIBRARIES})
```

Comment:
I'm not sure anymore if we did this on purpose because Boost needed to be linked before some other when statically linking or so. Is there a reason you are changing this here or is it only for alphabetical reasoning? Because if it's the latter I wouldn't change it.

Reply:
Ditto for https://github.com/SiegeEngineers/auto-mods/pull/5#discussion_r566525576